openmm_septop/python_tutorial.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>I think you're mentioning the wrong Protocol here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>